### DISCUSSION TITLE
How do we enable AddressSanitizer option?

### DISCUSSION THREAD

**Discussion by CybernetHacker14** [premake/premake-core#1938] [Q&A] [answered]
In MSVC, there's an option to enable AddressSanitizer in C++ projects.
![Screenshot_1](https://user-images.githubusercontent.com/26383905/187168987-7d628f00-e1d8-48be-8d91-1dd7634f5ffd.png)
This is available natively in Visual Studio 2022, as far as I know. I wanted to know how do we enable this option by adding it to a premake script, since the entire solution is generated using premake.

**Comment by nickclark2016** [comment, marked as answer]
We've currently got a PR being worked on for this functionality.

**Comment by Jarod42** [comment]
BTW, `buildoptions("/fsatinize=address")` should do the job, except that the UI won't show it is activated (would be visible in Other options).
